client/actions/cycles.js

```diff
@@ -116,3 +140,73 @@ export function deleteUsersFromRun(users, runId, cycleId) {
         dispatch(deleteUsersFromRunDispatch({ ...response.data, cycleId }));
     };
 }
+
+/**
+ * getIssuesByTestId        Retrieve all issues known to ARIA-AT and
+ *                          their corresponding Github Issue object
+ * @param  {Number} test_id The test_id that corresponds to a given Issue
+ * @return {Array}          An array of Github Issue objects.
+ */
+export function getIssuesByTestId(test_id) {
+    return async function(dispatch) {
+        const response = await axios.get(
+            `/api/cycle/issues?test_id=${test_id}`
+        );
+        dispatch(getIssuesByTestIdDispatch(response.data));
+    };
+}
+
+/**
+ * createIssue           Create a new issue via Github API
+ * @param  {Object} data Title, body, run, test
+ * @return {Issue}       The Issue object returned by Github
+ */
+export function createIssue(data) {
+    return async function(dispatch) {
+        const response = await axios.post('/api/cycle/issue', {
+            data
+        });
+        dispatch(createIssueSuccessDispatch(response.data));
+    };
+}
+
+//
+// TODO: These are intentionally unimplemented and will
+//       be completed in a follow up changeset.
+//
+
+/**
+ * getConflictsByCycleId        Returns an array of conflicts for a
+ *                              test in a given cycle.
+ * @param  {Number} cycle_id    The id of the cycle to retrieve conflicts
+ * @return {Array}              An array of conflict objects.
+ */
+export function getConflictsByCycleId(cycle_id) {
```

Comment:
We actually don't need this. All the results for a cycle are already in the front end -- so we can do the conflict calculation on the front end.

As it is right now, having all test results for all test for a cycle when you contribute to a cycle -- might seems a little heavy handed.. but for now it is not too much data. we might need to later serve things in smaller pieces.

Reply:
We might be thinking of something different here. The conflicts/diffs that @s3ththompson and I planned out are based on a serialization of the actual form itself within the iframe. That would also be used to repopulate those forms the next time you open the test to run it. The current data that I saw in the result objects does not have the details necessary to do that.

Reply:
i think the issue is that there are 2 concepts here with similar terminology. here's my conception of them, hopefully it matches yours?
- [1] the [results conflict diff](https://github.com/w3c/aria-at/issues/132#issuecomment-617225005), which Matt sketched out, which will go at the bottom of the GitHub issue, which Val points out can be generated on the frontend.
- [2] the (basically) vdom state that will need to be serialized from the iframe form, saved to the db, and then retrieved to re-hydrate the form during editing.

@rwaldron it sounds like this stub is for [2]? if so, is it appropriately named? don't we need to save the serialized data for all results, whether it's conflicted or not?

Reply:
@s3ththompson yes, you're correct. @spectranaut and I had a call to discuss this and I described our idea and she's got it now.